api/rsi.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+import "base.proto";
+
+package rsi;
+
+option go_package = "internal/proto/monitor";
+
+service rsi {
+  rpc get_rsi (PriceData) returns (RsiData);
```

Comment:
according to https://developers.google.com/protocol-buffers/docs/style you "should use CamelCase (with an initial capital) for both the service name and any RPC method names"

Reply:
changed from rust convention to protobuf style in #cc7e1c0ebc0a944733fe9516576a0339bce09b3f

Reply:
#cc7e1c0

Reply:
:)